Scripts/Src/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.IO;
+
+namespace bootstrap_script
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            string AppName = args[0];
+            Console.WriteLine("Setting up bootstrap for " + AppName);
+            string BootstrapRootDir = "";
+            string BootstrapName = "NetCoreBootstrap";
+            foreach (var Dir in Directory.GetCurrentDirectory().Split('/'))
+            {
+                BootstrapRootDir += Dir + "/";
+                if (Dir == AppName) break;
+            }
+            Console.WriteLine("Replacing " + BootstrapName + " to " + AppName + " in " + BootstrapRootDir);
+            string Contents = "";
+            foreach (string file in Directory.EnumerateFiles(BootstrapRootDir, "*", SearchOption.AllDirectories))
+            {
+                if (file.Contains("bootstrap-script"))

```

Comment:
what's the most used practice regarding wether to leave or remove brackets in this kind of control statements?